gradle/dependency-locks/default.lockfile

```diff
@@ -24,8 +25,9 @@ net.bytebuddy:byte-buddy:1.7.9
 org.apache.commons:commons-compress:1.9
 org.apache.commons:commons-csv:1.3
 org.apache.commons:commons-exec:1.3
-org.apache.commons:commons-lang3:3.5
+org.apache.commons:commons-lang3:3.9
 org.apache.commons:commons-math3:3.6.1
+org.apache.commons:commons-text:1.8
```

Comment:
Kotlin stdlib offers a lot of text functions, e.g. [`String.capitalize`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/capitalize.html).